help/include/lists.md

```diff
@@ -0,0 +1,33 @@
+## Bulleted list syntax
+
+### Example
```

Comment:
Let's add a bit of an intro to how these instructions work. Something like:

Here is an example of how you can use bulleted lists in Zulip.

And then maybe "What you type" and "What you get" for section titles (or something like that); I think it sounds too much like programmer language currently. :)

(same feedback applies to numbered lists, of course)

Reply:
> And then maybe "What you type" and "What you get" for section titles (or something like that); I think it sounds too much like programmer language currently. :)

Yeah, good point. I see what you are saying. :)

Reply:
Maybe "What you type" and "What it looks like"?

Reply:
<img width="500" alt="image" src="https://user-images.githubusercontent.com/2343554/218235291-a83cd52c-4d27-45c9-9910-24f6c6ed8381.png">